package.json

```diff
@@ -57,6 +57,7 @@
     "ember-cli": "2.5.1",
     "ember-cli-app-version": "^1.0.0",
     "ember-cli-dependency-checker": "^1.2.0",
+    "ember-cli-fastboot": "1.0.0-beta.9",
```

Comment:
sorry to be pedantic, but anything that isn't in the ember-cli app  blueprint package.json is better to be down the bottom list. So this means ember-cli-fastboot, ember-cli-fastboot-addon-tests and ember-sinon

https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/package.json

makes upgrading much simpler.